examples/websocket/server/websocket.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/gobwas/ws"
+	"github.com/gobwas/ws/wsutil"
+	"github.com/panjf2000/gnet/v2"
+	"github.com/panjf2000/gnet/v2/pkg/logging"
+	"log"
+	"sync/atomic"
+	"time"
+)
+
+type wsServer struct {
+	*gnet.BuiltinEventEngine
+
+	addr      string
+	multicore bool
+	eng       gnet.Engine
+	connected int64
+}
+
+type wsCodec struct {
+	ws bool
+}
+
+func (wss *wsServer) OnBoot(eng gnet.Engine) gnet.Action {
+	wss.eng = eng
+	log.Printf("echo server with multi-core=%t is listening on %s\n", wss.multicore, wss.addr)
```

Comment:
Please use gnet `logging` instead of standard `log`, replace all usages in this PR.

Reply:
<img width="909" alt="image" src="https://user-images.githubusercontent.com/13690966/153593227-023c6bde-f2fa-4a81-9567-09ad9fb57364.png">

<img width="1186" alt="image" src="https://user-images.githubusercontent.com/13690966/153593259-264b6b87-0eb2-4970-bf0d-fc3bcefba9b8.png">